### PR TITLE
#99 Add ability to connect to an already existing instance of Chrome

### PR DESCRIPTION
**Pull request by gidztech**
See https://github.com/smooth-code/jest-puppeteer/issues/99 for discussion.

### CODE REVIEW

File: packages/jest-environment-puppeteer/src/readConfig.js [hunk 1]
```diff
@@ -38,7 +38,13 @@ async function readConfig() {
   }
 
   // eslint-disable-next-line global-require, import/no-dynamic-require
-  const localConfig = require(absConfigPath)
+  const requiredConfig = require(absConfigPath)
```
Comment (neoziro) [inline packages/jest-environment-puppeteer/src/readConfig.js]: Why should we support async config? I think it is another issue.

**Comment by gidztech** [comment]
The async function config I originally added was so that I could run an async task first (e.g. launching Chrome and get the WebSocket URI) before returning the `connect` object with the URI. However, I realised I could do that work in a Jest `setup` file, which executes beforehand. Therefore, adding an async function is no longer needed right now. 

I will update the PR this weekend.

**Comment by neoziro** [comment]
@gidztech OK thanks

**Comment by gidztech** [comment]
@neoziro I made some simplifications thanks to my forgotten friend "object spread"

**Comment by gidztech** [comment]
Let me know if there's anything else to do. It would be good to get this in.

**Comment by neoziro** [comment]
@gidztech good for me! Can you update readme please? 🙏

- https://github.com/smooth-code/jest-puppeteer/tree/master/packages/jest-environment-puppeteer/README.md
- https://github.com/smooth-code/jest-puppeteer/tree/master/README.md

**Comment by gidztech** [comment]
@neoziro Added documentation (with example usage)

**Pull request merged**